tests-legacy/Unit/Core/SpecificPrice/SpecificPriceFormatterTest.php

```diff
@@ -24,39 +24,29 @@
  * International Registered Trademark & Property of PrestaShop SA
  */
```

Comment:
This is a legacy test that I moved in a new test file (since the code tested is in another class).
I just added some testing without taxes included.